next/utils/useAccount.tsx

```diff
@@ -74,6 +75,9 @@ const updatableAttributes = new Set([
 const poolData = {
   UserPoolId: process.env.NEXT_PUBLIC_COGNITO_USER_POOL_ID || '',
   ClientId: process.env.NEXT_PUBLIC_COGNITO_CLIENT_ID || '',
+  Storage: new CookieStorage({
```

Comment:
Would be nice to have func returning CookieStorage object.

Reply:
:+1: honestly, will again do with the other useAccount changes once I get to those